lib/logitech_receiver/settings_templates.py

```diff
@@ -189,6 +192,11 @@ def _feature_k375s_fn_swap():
 					label=_FN_SWAP[1], description=_FN_SWAP[2],
 					device_kind=(_DK.keyboard,))
 
+def _feature_backlight2():
+	return feature_toggle(_BACKLIGHT[0], _F.BACKLIGHT2,
```

Comment:
Please add the following in the line above.

```
# FIXME: This will enable all supported backlight settings, we should allow the users to select which settings they want to enable.
```

If you want to actually fix this then go to https://github.com/Logitech/cpg-docs and create an issue asking for the documentation for `0x1982`.

Reply:
Also, the current approach might have a side effect where it won't work in some devices, although I find it very unlikely.

Reply:
just realized that `hand_detection` setting is very similar or same (Turn on illumination when the hands hover over the keyboard.). i recommend to remove my `backlight` feature,and rename `_feature_backlight2` to `_feature_handdetection`, but keep the` _F.BACKLIGHT2`. your opinion?

Reply:
Then you maybe you just implement the correct spec. I will start writing the documentation to be published in the repo I linked.

Reply:
i think i dont get it.. in hid20 its called `backlight 2`,
for hid10 it is a register ``handdetection`  (https://github.com/pwr-Solaar/Solaar/blob/59c036728381beb1a121948f5e3ebf85bc41f3d0/lib/logitech_receiver/hidpp10.py#L121)

Reply:
I don't want to merge the code as it is right now. This hand detection stuff just makes things confusing.

> what do you mean with

The Backlight 2 feature is defined in the HID++ 2.0 spec, it export several settings. When you talk with the device you basically say that you want to configure the feature `0x1982` (Backlight 2) and pass some arguments. What I want to do it to actually support all the arguments that the specification allows. However I don't think I own any device with the `0x1982` feature so you will need to be the one adding that, I can help you though.

Reply:
i reverted last commit, but added the #FIXME and cleanup the device `craft`.

Reply:
Ok, thanks! I just want to confirm, this feature with your patch does work as intended right?